### PR TITLE
Installs Serverless operator 1.2.0

### PR DESCRIPTION
**Pull request by navidshaikh**
- setup Knative Serving v0.9.0
 - removes the manual install of deps for Serving

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -194,7 +195,7 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
+  oc apply -f openshift/serverless/serverless-operator-1.2.0.yaml -n openshift-operators
```
Comment (rhuss) [inline openshift/e2e-tests-openshift.sh]: Is there a way to refer to the yaml file directly from its release page instead of copying it into this repo ?

Comment (rhuss) [reply openshift/e2e-tests-openshift.sh]: Can we please parameterize the version number into a global variable

File: openshift/e2e-tests-openshift.sh [hunk 2]
```diff
@@ -305,10 +306,10 @@ failed=0
 
 (( !failed )) && build_knative_client || failed=1
 
-(( !failed )) && install_servicemesh || failed=1
-
 (( !failed )) && install_knative_serving || failed=1
 
+(( !failed )) && install_servicemesh || failed=1
+
```
Comment (rhuss) [inline openshift/e2e-tests-openshift.sh]: Have not dive into the script yet,  but not sure why the servicemesh should be installed after knative when knative would need the service mesh ?

File: openshift/e2e-tests-openshift.sh [hunk 3]
```diff
@@ -127,17 +127,18 @@ function timeout() {
 }
 
 function install_servicemesh(){
-  header "Installing ServiceMesh"
+  # header "Installing ServiceMesh"
 
   # Install the ServiceMesh Operator
-  oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/$SERVING_RELEASE/openshift/servicemesh/operator-install.yaml
+  # oc apply -f https://raw.githubusercontent.com/openshift/knative-serving/$SERVING_RELEASE/openshift/servicemesh/operator-install.yaml
```
Comment (rhuss) [inline openshift/e2e-tests-openshift.sh]: Can we remove it instead of commenting out ? Where is this operator now installed ?

**Comment by rhuss** [comment]
more questions than a review actually ;)

**Comment by navidshaikh** [comment]
@rhuss : Its more of an experimental PR than actual one, thus I removed reviewers to avoid spamming them ;-) I will close this.

**Pull request closed without merging**